src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int AMOUNT_OF_BALLS = 3;
+    private static final Lottery lottery = new Lottery();
+    private static final Ball[] balls = new Ball[AMOUNT_OF_BALLS];
```

Comment:
instantiate lottery and create ball array inside main method
```suggestion
    private static final Lottery lottery = new Lottery();
    private static final Ball[] balls = new Ball[AMOUNT_OF_BALLS];
```